### PR TITLE
Fix UMA criu enablement

### PR DESCRIPTION
**Pull request by tajila**
Fix UMA criu enablement

Signed-off-by: Tobi Ajila <atobia@ca.ibm.com>

### CODE REVIEW

File: closed/OpenJ9.gmk [hunk 1]
```diff
@@ -249,9 +249,9 @@ endif
 
 # Adjust CRIU Support enablement flags.
 ifeq (true,$(OPENJ9_ENABLE_CRIU_SUPPORT))
-  FEATURE_SED_SCRIPT += $(call SedEnable,opt_criuSupport)
+  SPEC_SED_SCRIPT += $(call SedEnable,opt_criuSupport)
 else # OPENJ9_ENABLE_CRIU_SUPPORT
-  FEATURE_SED_SCRIPT += $(call SedDisable,opt_criuSupport)
+  SPEC_SED_SCRIPT += $(call SedDisable,opt_criuSupport)
```
Comment (keithc-ca) [inline closed/OpenJ9.gmk:254]: This won't help because the file that needs to be modified is `f9.flags` which is neither `.feature` nor `.spec`.

Comment (tajila) [reply closed/OpenJ9.gmk:254]: Do you mean this? https://github.com/eclipse-openj9/openj9/blob/master/buildspecs/j9.flags#L1705

Comment (tajila) [reply closed/OpenJ9.gmk:254]: I guess I could leave it as is and add it to https://github.com/eclipse-openj9/openj9/blob/master/buildspecs/core.feature

Comment (keithc-ca) [reply closed/OpenJ9.gmk:254]: Yes, I think a default for the flag belongs in some `.feature` file, `core.feature` seems reasonable enough.

**Pull request closed without merging**